src/collar/oc_api.lsl

```diff
@@ -110,7 +110,7 @@ key g_kMenuUser;
 integer CalcAuth(key kID, integer iVerbose){
     string sID = (string)kID;
     // First check
-    if(llGetListLength(g_lOwner) == 0 && kID==g_kWearer && llListFindList(g_lBlock,[sID])==-1)
```

Comment:
Why would we not want to check the block list here?

Reply:
Because that is a separate issue.  This only addresses the range issue, per you, the pull only addresses what it says it does. 

The issue of wearer being able to be blocked, and being able to be trusted, needs to be handled differently.